nuxeo-jsf/nuxeo-platform-ui-web/src/test/java/org/nuxeo/ecm/platform/ui/web/restAPI/TestUpdateDocumentRestlet.java

```diff
@@ -77,6 +77,7 @@ protected void dotTestUpdate(String endpoint) throws Exception {
 
         // check doc has been updated
         txFeature.nextTransaction();
+        doc = session.getDocument(doc.getRef());
```

Comment:
Curious: why isn't this needed in master?

Reply:
It is: https://github.com/nuxeo/nuxeo-jsf-ui/pull/1